app/src/main/java/org/eyeseetea/malariacare/data/database/iomodules/dhis/importer/ConvertFromSDKVisitor.java

```diff
@@ -40,7 +40,8 @@
 import java.util.List;
 import java.util.Map;
 
-public class ConvertFromSDKVisitor implements IConvertFromSDKVisitor {
+public class ConvertFromSDKVisitor implements
```

Comment:
@xurxodev  Only to be consistent: 
it is not so important but can you remove "org.eyeseetea.malariacare.data.database.iomodules.dhis.importer." and "org.eyeseetea.malariacare.data.database.model.XXX" and use only one import in the entire class?
I think we are using the imports instead of the entire path except when the SDK model have the same name than our model and they are used in same class, but in the new sdk we dont have this problem.